### ISSUE TITLE
TODO

### ISSUE THREAD

**Issue by jlord** [jlord/hack-spots#24]
- [ ] add pagination
- [ ] update mapbox.js
- [ ] clean up jquery

**Comment by deltamualpha** [comment]
Personally, I didn't like clustering, so having a toggle for that in the map generating method would be :+1:

**Comment by jlord** [comment]
@deltamualpha, yeah actually I'm feeling the same way. Just took it off the list :sparkles:
